targets/zephyr/Makefile.zephyr

```diff
@@ -70,7 +70,7 @@ CONFIG_TOOLCHAIN_VARIANT = x86
 CPU = x86
 EXT_CFLAGS += -march=pentium
 EXT_CFLAGS += -mpreferred-stack-boundary=2 -mno-sse
-else ifeq ($(BOARD),qemu_cortex_m3)
+else ifeq ($(BOARD),$(filter $(BOARD),qemu_cortex_m3 frdm_k64f))
 CONFIG_TOOLCHAIN_VARIANT = arm
```

Comment:
I see, thanks for providing the the necessary Zephyr context. I agree that ideally the compiler flags should be specified in a single location for Zephyr and then just cascade down to JerryScript. Perfectly fine to address this in a later patch once Zephyr has the appropriate support in place :)